stories/form/form-range-slider/props-table.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { ApiTable, code } from "../../storybook-common/api-table";
+import { ApiTableSectionProps } from "../../storybook-common/api-table/types";
+import { SHARED_FORM_PROPS_DATA } from "../shared-props-data";
+
+const DATA: ApiTableSectionProps[] = [
+    {
+        name: "InputRangeSlider specific props",
+        attributes: [
+            {
+                name: "value",
+                description: "The input values",
+                propTypes: ["number[]"],
+            },
+            {
+                name: "min",
+                description: "The start of the range",
+                propTypes: ["number"],
+                defaultValue: "0",
+            },
+            {
+                name: "max",
+                description: "The end of the range",
+                propTypes: ["number"],
+                defaultValue: "100",
+            },
+            {
+                name: "step",
+                description:
+                    "The interval between the previous and next values",
+                propTypes: ["number"],
+                defaultValue: "1",
+            },
+            {
+                name: "numOfThumbs",
+                description: "the number of controls",
+                propTypes: ["number"],
+                defaultValue: "2",
+            },
+            {
+                name: "minRange",
+                description: "The minimum difference between values",
+                propTypes: ["number"],
+            },
+            {
+                name: "disabled",
+                description: "the number of controls",
+                propTypes: ["boolean"],
+            },
+            {
+                name: "readOnly",
+                description: "the number of controls",
+                propTypes: ["boolean"],
+            },
+            {
+                name: "colors",
+                description: "List of custom colors for each track segment",
```

Comment:
Perhaps want to add the requirement that the expected length is (number of values + 1)?